src/app/resources/resources.component.html

```diff
@@ -25,16 +25,16 @@
       </button>
       <button mat-button [disabled]="!selection.selected.length || selectedNotAdded === 0" (click)="libraryToggle(selection.selected, 'add')">
         <mat-icon aria-hidden="true" class="margin-lr-3">library_add</mat-icon><span i18n>Add to myLibrary</span>
-        <span *ngIf="selection?.selected?.length">({{selectedNotAdded}})</span>
+        <span *ngIf="selection?.selected?.length"> ({{selectedNotAdded}})</span>
```

Comment:
I think rather than adding the space it would be better to style with margin left by adding the class like
 `        <span *ngIf="selection?.selected?.length" class="margin-lr-3">({{selectedNotAdded}})</span>
`

Reply:
@yuviii For block elements or block like inline elements (like buttons), yes, but with inline elements of text spaces are better.